shared/src/main/scala/drt/shared/DisplayMinute.scala

```diff
@@ -0,0 +1,14 @@
+package drt.shared
+
+sealed trait DisplayMinute {
+  val minutes: Int
+  val display: String
+}
+
+case class MinuteAsNoun(minutes: Int) extends DisplayMinute {
```

Comment:
Nice :+1:

Reply:
Sorry actually these are named the wrong way around. Adjective is when it's followed by a subject and should always be singular. Noun is when it's on its own and only singular when the value is 1